### ISSUE TITLE
No response/403 for specifically paginated requests

### ISSUE THREAD

**Issue by nachtm** [NYTimes/public_api_specs#25]
Hi,

The following request gives no response (I've curl-ed and gone through the python requests library):
`https://api.nytimes.com/svc/search/v2/articlesearch.json?fq=persons%3A%28%22Obama%2C+Barack%22%29&page=11&api-key=xxxx`

The following request gives an html "403 forbidden" response:
`https://api.nytimes.com/svc/search/v2/articlesearch.json?fq=persons%3A%28%22Trump%2C+Donald+J%22%29&page=12&api-key=xxxx`

The following requests give valid, seemingly correct json responses:
`https://api.nytimes.com/svc/search/v2/articlesearch.json?fq=persons%3A%28%22Trump%2C+Donald+J%22%29&page=11&api-key=xxxx`

`https://api.nytimes.com/svc/search/v2/articlesearch.json?fq=persons%3A%28%22Trump%2C+Donald+J%22%29&page=13&api-key=xxxx`

`https://api.nytimes.com/svc/search/v2/articlesearch.json?fq=persons%3A%28%22Obama%2C+Barack%22%29&page=10&api-key=xxxx`

`https://api.nytimes.com/svc/search/v2/articlesearch.json?fq=persons%3A%28%22Obama%2C+Barack%22%29&page=12&api-key=xxxx`

In other words, the 11th page of `fq=persons:("Obama, Barack")` returns nothing (despite the 10th and 13th returning correct results), and the 12th page of `fq=persons:("Trump, Donald J")` returns a 403 Forbidden page (despite the 11th and 13th returning correct results).

A friend tried this on his computer with a different api key and got the same results. 

Any ideas on what's going on?

**Comment by nyt-hughmandeville** [comment]
This is a known bug we are working on fixing.  A work around is to set the field list in the query parameters to only the fields you want (e.g. fl=web_url).

**Comment by nachtm** [comment]
Thanks! I think that worked for me.

**Comment by nyt-hughmandeville** [comment]
The 403 bug should be fixed now.

**Comment by nachtm** [comment]
Yep, I can confirm that it's working for me. Thank you!
